src/CommonDBTM.php

```diff
@@ -5561,7 +5561,7 @@ private function assetBusinessRules($condition)
             $params = [
                 'condition' => $condition
             ];
-            $output = $ruleasset->processAllRules($input, [], $params);
+            $output = $ruleasset->processAllRules($input, [], [], $params);
```

Comment:
Could you rename `$params` to `$options`, in order to fits method declaration ?

Reply:
Done in [37f48a7](https://github.com/glpi-project/glpi/pull/11716/commits/37f48a74554c450be2d6b2f72f568846ac05a7ac), I've removed the variable as it is only used once.